### DISCUSSION TITLE
Proposal: Further enhance accuracy

### DISCUSSION THREAD

**Discussion by Kornelius777** [jomjol/AI-on-the-edge-device#1238] [Q&A]
This little tool is already doing a nice job - however,
on my water meter, I have four analogue readings (for 0.1, 0.01, 0.001 and 0.0001 m³).
It occurs that when e.g. the "0.001" reading is somewhere between 8 and 9, the "0.01" reading suddenly jumps to the next number, because it is getting too close to the next digit.
My I suggest to implement an additional sanity check? 

It should be like "if the reading of the following digit is somewhere in front of the "0" (meaning 7...9), do not assume the preceding digit having changed its value"

Do you think, this could be implemented?

Kind regards!

**Comment by mad2xlc** [comment]
Hi, try it with the setting CheckDigitIncreaseConsistency in the expert settings. This should help.
